e2e/__snapshots__/templates.test.js.snap

```diff
@@ -3451,7 +3451,7 @@ If you want to publish it as a public scoped package, run \`npm publish --access
 exports[`Templates InstantSearch.js widget File content: LICENSE.md 1`] = `
 "The MIT License (MIT)
 
-Copyright (c) 2021-present algolia.
+Copyright (c) 2022-present algolia.
```

Comment:
Why change this to 2022? Shouldn't it stay the start date of this license (hence the "-present")?

Reply:
It doesn't reference the start date. The template looks like this:

```hbs
Copyright (c) {{ currentYear }}-present {{ organization }}.
```

It kind of makes sense as it sets the creation date of template-generated project.

Reply:
Yes sorry, I didn't notice the context of the test.

We might want to use `new Date().getFullYear()` at some point to avoid changing this every year.